spec/geckoboard/client_spec.rb

```diff
@@ -149,10 +153,39 @@ def make_request
           end
         end
 
+        context 'with the optional `unique_by` parameter' do
+          include_examples :bad_response_exceptions
+
+          let(:unique_by_array) { ['datetime'] }
```

Comment:
🏄  Is `unique_by` already taken in this context? the "array" part feels a bit redundant